clap_complete/examples/completion_subcommands.rs

```diff
@@ -0,0 +1,41 @@
+//! Example to test subcommands.
+//!
+//! Usage with bash:
+//! ```sh
+//! cargo run --example completion_subcommands -- --generate=bash > completion_subcommands.bash
+//! . ./completion_subcommands.bash
+//! ./target/debug/examples/completion_subcommands <TAB>
+//! ./target/debug/examples/completion_subcommands help <TAB>
+//! ```
+use clap::{value_parser, Arg, Command};
+use clap_complete::{generate, Generator, Shell};
+use std::io;
+
+fn build_cli() -> Command<'static> {
```

Comment:
Could you remove this example?

I'd prefer examples to focus on how to use clap_complete and not just to demonstrate some desired functionality works (thats what tests are for)